channels/dz.m3u

```diff
@@ -1,49 +1,7 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlAnisTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/G60iJen.png" group-title="",Al Anis TV [Offline]
-https://cdn02.hta.dz/abr_htatv/El_Fhama_TV/playlist.m3u8
-#EXTINF:-1 tvg-id="AlgerieTV3.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/J6VmBuw.png" group-title="",Algérie TV3 [Offline]
-https://cdn02.hta.dz/abr_htatv/A3_HD/playlist.m3u8
-#EXTINF:-1 tvg-id="AlgerieTV4.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/n8httHt.png" group-title="",Algérie TV4 [Offline]
-https://cdn02.hta.dz/abr_htatv/TV_4/playlist.m3u8
-#EXTINF:-1 tvg-id="AlgerieTV5.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/gXG15AX.png" group-title="",Algérie TV5 [Offline]
-https://cdn02.hta.dz/abr_htatv/TV_5/playlist.m3u8
-#EXTINF:-1 tvg-id="AlgerieTV6.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/HYqJY6y.png" group-title="",Algérie TV6 [Offline]
-https://cdn02.hta.dz/abr_htatv/TV_6_HD/playlist.m3u8
-#EXTINF:-1 tvg-id="BahiaTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oBRXuH2.png" group-title="",Bahia TV [Offline]
-https://cdn02.hta.dz/abr_htatv/Bahia_TV/playlist.m3u8
-#EXTINF:-1 tvg-id="CanalAlgerie.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/FPmDhNH.png" group-title="",Canal Algérie [Offline]
-https://cdn02.hta.dz/abr_htatv/CANAL_ALGERIE/playlist.m3u8
-#EXTINF:-1 tvg-id="CNA.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/4s1NlRf.jpg" group-title="Religious",CNA (Chaîne Nord Africaine) (360p)
-https://live.creacast.com/cna/smil:cna.smil/playlist.m3u8
```

Comment:
We do not remove broken links from playlists if they have no alternatives to keep the channel description (logo, language, country)